bundler.go

```diff
@@ -73,6 +74,9 @@ type Configuration struct {
 
 	//!\\ DEBUG ONLY
 	AstilectronPath string `json:"astilectron_path"` // when making changes to astilectron
+
+	// LDFlags to pass through to go build
+	LDFlags
```

Comment:
I'd rather you don't use an anonymous field here. And I'd rather you add a `json` tag as well.

Therefore something along the lines of:

```go
LDFlags LDFlags `json:"ldflags"`
```